rgw/v2/tests/s3_swift/test_user_bucket_rename.py

```diff
@@ -1,4 +1,16 @@
-# test user and bucket rename with tenanted and non tenanted users
+"""test_user_bucket_rename - Test with Tenanted and Non-Tenanted User
+Usage: test_user_bucket_rename.py [options] <input>
+Options:
+    -c=<configs>    Configs [default: test_user_bucket_rename.yaml]
```

Comment:
there is no default file. you need to explicitly mention the yaml for `-c`. if there are more than one list of yamls, please add them. 
probably you call follow this structure: 
```
"""test_versioning_copy_objects - Test Copy of objects with Versioned and non-versionsed buckets.

Usage: test_versioning_copy_objects.py -c <input_yaml>

<input_yaml>
        Note: any one of these yamls can be used
	config1.yaml 
        config2.yaml


Operation:
	Create a bucket and a versioned bucket
 	Copy objects between buckets 
	verify version ID is created for objects in versioned buckets.

"""
```

@mkasturi18  has a similar structure like above. gives more meaning